docs/en/04-supported-codebases/mmdet.md

```diff
@@ -1,30 +1,175 @@
+    faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth \
+    tests/data/tiger.jpeg \
+    --work-dir mmdeploy_models/mmdet/ort \
+    --device cpu \
+    --show \
+    --dump-info
+```
+
+It is crucial to specify the correct deployment config during model conversion. We've already provided builtin deployment config [files](https://github.com/open-mmlab/mmdeploy/tree/dev-1.x/configs/mmdet) of all supported backends for mmdetection, under which the config file path follows the pattern:
+
+```
+{task}/{task}_{backend}-{precision}_{static | dynamic}_{shape}.py
+```
+
+- **{task}:** task in mmdetection.
+
+  There are two of them. One is `detection` and the other is `instance-seg`, indicating instance segmentation.
+
+  mmdet models like `RetinaNet`, `Faster R-CNN` and `DETR` and so on belongs to `detection` task. While `Mask R-CNN` is one of `instance-seg` models. You can find more of them in chapter [Supported models](#supported-models).
+
+  **DO remember to use** `detection/detection_*.py` deployment config file when trying to converting detection models and use `instance-seg/instance-seg_*.py` to deploy instance segmentation models.
+
+- **{backend}:** inference backend, such as onnxruntime, tensorrt, pplnn, ncnn, openvino, coreml and etc.
+
+- **{precision}:** fp16, int8. When it's empty, it means fp32
+
+- **{static | dynamic}:** static shape or dynamic shape
+
+- **{shape}:** input shape or shape range of a model
+
+Therefore, in the above example, you can also convert `faster r-cnn` to other backend models by changing the deployment config file `detection_onnxruntime_dynamic.py` to [others](https://github.com/open-mmlab/mmdeploy/tree/dev-1.x/configs/mmdet/detection), e.g., converting to tensorrt-fp16 model by `detection_tensorrt-fp16_dynamic-320x320-1344x1344.py`.
+
+```{tip}
+When converting mmdet models to tensorrt models, --device should be set to "cuda"
+```
+
+## Model Specification
+
+Before moving on to model inference chapter, let us talk more about the converted model structure which is very important to do model inference.
+
+The converted model locates in the working directory like `mmdeploy_models/mmdet/ort` in the previous example. It includes:
+
+```
+mmdeploy_models/mmdet/ort
+├── deploy.json
+├── detail.json
+├── end2end.onnx
+└── pipeline.json
+```
+
+in which,
+
+- **end2end.onnx**: backend model which can be inferred by ONNX Runtime
+- **deploy.json**: meta information about backend model
+- **pipeline.json**: inference pipeline of mmdeploy SDK
+- **detail.json**: conversion parameters
+
+And the whole package **mmdeploy_models/mmdet/ort** is defined as **mmdeploy SDK model**. In other words, **mmdeploy SDK model** includes not only backend model but also inference meta information.
+
+## Backend model inference
+
+mmdeploy provides a unified API named as `inference_model` to do this job, making all inference backends API transparent to users.
+
+Take the previous converted `end2end.onnx` model as an example,
```

Comment:
Take ..... as an example, you can use the following code to inference the model and visualize the results.